asciidoctor-revealjs.gemspec

```diff
@@ -0,0 +1,31 @@
+# -*- encoding: utf-8 -*-
+require File.expand_path '../lib/asciidoctor-revealjs/version', __FILE__
+
+Gem::Specification.new do |s|
+  s.name = 'asciidoctor-revealjs'
+  s.version = Asciidoctor::Revealjs::VERSION
+  s.authors = ['Olivier Bilodeau']
+  s.email = ['olivier@bottomlesspit.org']
+  s.homepage = 'https://github.com/asciidoctor/asciidoctor-reveal.js'
+  s.summary = 'Converts AsciiDoc to HTML for a Reveal.js presentation'
+  s.description = 'An Asciidoctor converter that generates the HTML component of a Reveal.js presentation from AsciiDoc.'
+  s.license = 'MIT'
+  s.required_ruby_version = '>= 1.9.3'
+
+  files = begin
+    IO.popen('git ls-files -z') {|io| io.read }.split "\0"
+  rescue
+    Dir['**/*']
+  end
+  s.files = files.grep(/^(?:(?:bin|lib|templates)\/.+|Rakefile|(LICENSE|README)\.adoc)$/)
```

Comment:
I've updated the logic for selecting files for the gem that is compatible across all Ruby platforms.

See https://github.com/asciidoctor/asciidoctor-bespoke/blob/6c24942259891819745f3a2f9dcc4828e6ba1fab/asciidoctor-bespoke.gemspec#L15-L20